ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -52,25 +66,43 @@ public function transform($nodeGroupRole)
     }
 
     /**
-     * @param FacadeInterface             $facade
-     * @param NodeGroupRoleInterface|null $source
+     * @param FacadeInterface     $nodeRoleFacade
+     * @param GroupInterface|null $group
      *
      * @throws RoleNotFoundHttpException
-     * @return mixed
+     * @throws TransformerParameterTypeException
+     *
+     * @return null|NodeGroupRoleInterface
      */
-    public function reverseTransform(FacadeInterface $facade, $source = null)
+    public function reverseTransform(FacadeInterface $nodeRoleFacade, $group = null)
```

Comment:
I think this will introduce a larger problem. If an integrator create it's own ``NodeGroupRoleTransformer``, he may create it with only 2 parameters. But the ``GroupTransformer`` will still call the ``NodeGroupRoleTransformer->reverseTransform`` with 3 parameters, leading to an exception.